site/_sprints/218.md

```diff
@@ -1,8 +1,8 @@
 ---
 title: Sprint 218 Details
 sprint_number: 218
-slides:
-recording:
+slides: https://www.slideshare.net/ManageIQ/sprint-217
```

Comment:
the slides name doesn't match... 217 vs. 218

Reply:
oh weird - I remember fixing this - wonder if I forgot to hit save

Reply:
Fixed.